__tests__/src/auth/router/v2.test.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+
+const supertest = require('supertest');
+const { server } = require('../../../../src/server');
+const { db, users } = require('../../../../src/auth/models/index');
+const request = supertest(server);
+
+let adminTest;
+let testUser = {};
+
+beforeAll(async () => {
+  await db.sync();
+  adminTest = await users.create({
+    username: 'triciaS',
+    password: 'passwordy',
+    role: 'admin',
+  });
+});
```

Comment:
You created an admin user, but didn't actually create the testUser in the database.